### DISCUSSION TITLE
How to allow download of file in shared items

### DISCUSSION THREAD

**Discussion by botzill** [directus/directus#15480] [Archive]
Hi.

When I share an item that has a file field, it's not possible to download it in shared

<img width="1021" alt="Screenshot 2022-09-08 at 14 17 31" src="https://user-images.githubusercontent.com/4686040/189108995-0b8d8256-c1ef-457f-9794-fe5effea3ccd.png">

Is there a way to allow this download?

**Comment by rijkvanzanten** [comment]
@botzill Does the role you used for the share have access to directus files?

**Reply by botzill** [reply]
yes, when I login with that user I do have access to download file.

<img width="985" alt="Screenshot 2022-09-13 at 12 17 42" src="https://user-images.githubusercontent.com/4686040/189863263-b35faa03-4c24-4e22-8b12-e04d69987b55.png">

But I try this shared link from a non logged in account, should it work ?

**Reply by rijkvanzanten** [reply]
When you share this item, did you select a role that the share would use? If not, it will default to the public role permissions, which may or may not have access to directus files 🙂

**Reply by botzill** [reply]
Yes I do.
